ajax/condition.php

```diff
@@ -32,6 +32,31 @@
 include ('../../../inc/includes.php');
 Session::checkRight('entity', UPDATE);
 
+// integrity check
+if (!isset($_POST['itemtype']) || !isset($_POST['items_id'])) {
+   http_response_code(400);
+   die();
+}
+if (!is_subclass_of($_POST['itemtype'], PluginFormcreatorConditionnableInterface::class)) {
+   http_response_code(400);
+   die();
+}
+
+// Build an empty item or load it from DB
+/** @var CommonDBTM $parent */
+$parent = new $_POST['itemtype'];
+if ($parent::isNewID((int) $_POST['items_id'])) {
+   $parent->getEmpty();
+   $parent->fields = array_intersect_key($_POST, $parent->fields);
+} else {
+   if (!$parent->getFromDB((int) $_POST['items_id'])) {
+      http_response_code(404);
+      die();
+   }
+}
+
 // get an empty condition HTML table row
 $condition = new PluginFormcreatorCondition();
-echo $condition->getConditionHtml($_POST);
+$condition->fields['itemtype'] = $_POST['itemtype'];
+$condition->fields['items_id'] = (int) $_POST['items_id'];
+echo $condition->getConditionHtml($parent);
```

Comment:
Atempt to simplify conditions creation and display for all conditionnable items. As conditions may be added to non-existing items (creating items and conditions at the same time), we need to forge items to make things work properly.

In other cases, loading them from DB is fine.

The most important need is to be able to find the form on top of the items, which are themselve parent of the conditions to show.